web/src/features/moreCast2/interfaces.ts

```diff
@@ -143,3 +143,8 @@ export interface ForecastMorecast2Row extends BaseRow {
   windDirection?: PredictionItem
   windSpeed?: PredictionItem
 }
+
+export interface DraftMorecast2Rows {
+  rows: MoreCast2Row[]
+  lastEdited: string | null
```

Comment:
In my experience I find it better to handle dates as objects and leave them as objects right up until the point they are displayed in the UI. Once you start working with dates as strings you can run into all sorts of pain related to timezones. 

I'd recommend that `lastEdited` be a luxon DateTime.

Reply:
Happy to discuss though!